docs/how-to/build-a-12-factor-app-rock.md

```diff
@@ -0,0 +1,157 @@
+# How to build a 12-Factor app rock
+
+## Include extra files in the OCI image
+
+The following files are included in the image by default from the root of the project:
+
+- `app` (does not apply to the `go-framework`)
+- `app.py` (does not apply to the `go-framework`)
+- `migrate`
+- `migrate.sh`
+- `migrate.py` (does not apply to the `go-framework`)
+- `static`
+- `templates`
+
+To change this, the following snippet needs to be added to the `rockcraft.yaml`:
+
+.. tabs::
```

Comment:
these are not rendering as expected:
![image](https://github.com/user-attachments/assets/69b75858-7e21-4602-9b58-169a68c3530e)

Reply:
Thanks for pointing this out. I realize now that I was not even using the correct file type (`.md` instead of `.rst`) and that was the main reason why the rendering was so bad. The tabs are rendering much more nicely now -- please take a look and let me know if you have more comments.